src/app/home/header/header.component.html

```diff
@@ -0,0 +1,13 @@
+<ion-header class="home-header">
```

Comment:
Shouldn’t this be separated in different shared header (only part of it) component for the whole app?

Reply:
agree